layouts/shortcodes/changelog/changelogs.html

```diff
@@ -1,10 +1,8 @@
 {{/*
-   This template is used to render all of the changelog sections under
-   "content/changelogs"
+   This template is used to redirect the changelog section under
+   "content/changelogs" to the latest version's changelog page.
 */}}
 
-{{ with .Page.Resources.Match "*.md" }}
-{{ range ((sort . "Params.date" "desc")) }}
-{{ .RenderShortcodes }}
-{{ end }}
+{{ with index .Page.RegularPages.ByDate.Reverse 0 }}
+   <meta http-equiv="refresh" content="0; url={{ .Permalink }}" />
 {{ end }}
```

Comment:
we could add something like:

```yaml
manualLinkRelRef: unstable.md
```

to the front matter of `content/changelog/_index.html`, but it would need maintaining during releases. (We could hack the `generate-changelog.sh` script to update it?)

Or we could add more intelligence to `layouts/partials/sidebar-tree.html` to handle this case?

This is probably fine for now, and we could improve it in a future PR.